.circleci/src/jobs/check_backend.yml

```diff
@@ -0,0 +1,32 @@
+environment:
+  _JAVA_OPTIONS: "-Xmx3g"
+  GRADLE_OPTS: "-Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2"
+docker:
+  - image: cimg/openjdk:17.0.6-node
+working_directory: ~/project/backend
+steps:
+  - checkout:
+      path: ~/project
+  - run: git submodule sync
+  - run: git submodule update --init
+  - restore_cache:
+      key: v2-gradle-wrapper-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+  - restore_cache:
+      key: v2-gradle-cache-{{ checksum "build.gradle.kts" }}
+  - run:
+      name: Check that GraphQL scheme is stable
+      command: |
+        ./gradlew run --args="graphql-export ../specs/backend-api.graphql"
+        git diff --exit-code
+  - run:
+      name: Test
+      command: |
+        ./gradlew test
+  - save_cache:
+      paths:
+        - .gradle/wrapper
+      key: v2-gradle-wrapper-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+  - save_cache:
+      paths:
+        - .gradle/caches
```

Comment:
These two paths (`.gradle/wrapper` and `.gradle/caches`)are not set correctly. You have set the working direcotry for the whole job to `~project/backend` and the path for cache path is relative to the working_directory of your job. But these .gradle paths are in the home directory

Reply:
the working directory is already set to this for the whole job. so it should be sufficient to change the path to
```js
 ~/gradle/wrapper
~/gradle/caches
```

Reply:
yes, but you need an dot before gradle, it should be `.gradle/wrapper`

Reply:
i adjusted it